app/services/category_service.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CategoryService
+  def self.conn
+    Faraday.new(
+      url: 'https://api.boardgameatlas.com/api/',
+      params: { client_id: ENV['CLIENT_ID'] }
+    )
+  end
+
+  def self.category
+    response = conn.get('game/categories') 
+    json = JSON.parse(response.body, symbolize_names: true)
+    json[:categories]
+  end
```

Comment:
Nice job here with querying the API. This might be able to be refactored into the current service BoardGameService as another method as we have some duplication with the conn method.